### ISSUE TITLE
Adding intial liquidity could target a price, instead of take inputs

### ISSUE THREAD

**Issue by Alexangelj** [primitivefinance/primitive-interface#401]
![image](https://user-images.githubusercontent.com/38409137/103374486-90ca8380-4a8c-11eb-8dc1-babaa82b975d.png)

We could have a slider for "implied option price" that will set the inputs for the user. I've found myself doing this manually (changing inputs until the desired price is hit).

**Comment by zachdt** [comment]
I agree, maybe a slider for price and a single input for underlying?
